pom.xml

```diff
@@ -175,7 +175,7 @@
           <version>1.6.13</version>
           <extensions>true</extensions>
           <configuration>
-            <serverId>ossrh</serverId>
+            <serverId>sonatype-nexus-staging</serverId>
```

Comment:
I'd keep `ossrh` in all three instances, unless there is a reason to change repository ID.